client/src/components/card-list/card-list.tsx

```diff
@@ -30,21 +31,20 @@ const CardList = (props: CardListProps) => {
 		const isPicked = !!picked?.find((pickedCard) => equalCard(card, pickedCard))
 		const isDisabled = !!disabled?.find((id) => card.cardId === id)
 
-		return (
-			<CSSTransition
-				timeout={250}
-				key={card.cardInstance}
-				unmountOnExit={true}
-				classNames={{
+		const cssClasses = [true, undefined].includes(enableAnimations)
+			? {
 					enter: css.enter,
 					enterActive: css.enterActive,
 					enterDone: css.enterDone,
 					exit: css.exit,
 					exitActive: css.exitActive,
-				}}
-			>
+			  }
+			: {}
+
+		return (
+			<CSSTransition timeout={250} unmountOnExit={true} classNames={cssClasses}>
 				<CardComponent
-					key={card.cardInstance}
+					key={card.cardId}
```

Comment:
I believe this would cause an error when there a multiple of the same card in one list.

Reply:
Fixed!